main.scss

```diff
@@ -13,6 +13,8 @@
 	@import "src/scss/print";
 	@import "src/scss/search";
 	@import "src/scss/tools";
+	@import "src/scss/themes/dark-on-light";
```

Comment:
Lets have `light` for the pink one, and then the next theme to be implemented (the transparent one for use in big photo-splash projects can be `transparent`